lib/blob_store/src/tracker.rs

```diff
@@ -287,11 +288,24 @@ impl Tracker {
     }
 
     /// Unset the value at the given point offset and return its previous value
-    pub fn unset(&mut self, point_offset: PointOffset) -> Option<ValuePointer> {
+    pub fn unset(
+        &mut self,
+        point_offset: PointOffset,
+        hw_counter: &HardwareCounterCell,
```

Comment:
Why only `unset` has `hw_counter` and `set` does not?

Reply:
The place where `set` is called is already measured, `unset` can only be measured here. I tried to measure things as early as possible in the call stack to avoid too many function changes.
However thinking about it again it might be beneficial to measure functions like `set` too, in order to not forget measurements later.

Reply:
Changed!